### ISSUE TITLE
Mismatched licenses

### ISSUE THREAD

**Issue by patrickplynch** [kowalpy/Robot-Framework-FTP-Library#27]
Hello all,

As per the license file the code is licensed as LGPL-3.0. However in multiple different files there are GPL license references. Is this a carryover from this issue https://github.com/kowalpy/Robot-Framework-FTP-Library/issues/12? Or is this library actually GPL? Thanks for your time!

# Locations of GPL referencing:

In the setup.py file at lines:
-  3-14: https://github.com/kowalpy/Robot-Framework-FTP-Library/blob/5016947da635c0739ab9bbc99abf98dfaadae8c0/setup.py#L1-L14
- 30: https://github.com/kowalpy/Robot-Framework-FTP-Library/blob/5016947da635c0739ab9bbc99abf98dfaadae8c0/setup.py#L30

In the FtpLibrary.py file at lines:
- 3-14: https://github.com/kowalpy/Robot-Framework-FTP-Library/blob/5016947da635c0739ab9bbc99abf98dfaadae8c0/FtpLibrary.py#L3-L14

**Comment by kowalpy** [comment]
Good catch @patrickplynch . I guess file comments should be updated so that all points to LGPL. This library is LGPL.

**Comment by patrickplynch** [comment]
Hey @kowalpyno problem, and that's good to hear! I was worried that this was actually licensed under GPL.

**Comment by MrMino** [comment]
I'm not a lawyer, and I hope I'm wrong, but...

I'm not sure if you can change the license like that. You don't own the right to relicense the work of other contributors - they still hold the copyright to their work.

The common understanding and [the GitHub ToS](https://github.com/github/site-policy/blob/1263bf0164cfe60d0900e70ba9126855c7c23c6c/Policies/github-terms-of-service.md#6-contributions-under-repository-license) is that the contributions are licensed under the license in the repository _at the time of the contribution_. This is normally called `inbound=oundbound norm`.

> Whenever you make a contribution to a repository containing notice of a license, you license your contribution under the same terms, and you agree that you have the right to license your contribution under those terms. If you have a separate agreement to license your contributions under different terms, such as a contributor license agreement, that agreement will supersede.

I believe this means that all contributions of persons other than @kowalpy made before 24753e73a9455a3f2f8430b1f4a0f3eadfad0a21 are still GPLv3.

I base this belief on the fact that if the license in a project could be changed this way, the strong copyleft guarantees of GPL licenses could be easily circumvented by a new maintainer just changing the license to MIT.
